tools/AutoTuner/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rayproject/ray:1.11.0
+
+RUN pip install -U ax-platform hyperopt nevergrad optuna pandas
```

Comment:
do we need a package for "random" search" as well?

Reply:
Random is included with Ray